src/__tests__/role-helpers.js

```diff
@@ -0,0 +1,120 @@
+import {getRoles, logRoles} from '../role-helpers'
+
+function setup() {
+  const section = document.createElement('section')
+  const h1 = document.createElement('h1')
+  const h2 = document.createElement('h2')
+  const h3 = document.createElement('h3')
+  const nav = document.createElement('nav')
+  const article = document.createElement('article')
+  const ul = document.createElement('ul')
+  const li1 = document.createElement('li')
+  const li2 = document.createElement('li')
+  const table = document.createElement('table')
+  const tr = document.createElement('tr')
+  const td1 = document.createElement('td')
+  const td2 = document.createElement('td')
+  const td3 = document.createElement('td')
+  const formEl = document.createElement('form')
+  const input = document.createElement('input')
+  const input2 = document.createElement('input')
+
+  section.appendChild(h1)
+  section.appendChild(nav)
+  section.appendChild(h2)
+  section.appendChild(h3)
+  section.appendChild(article)
+  article.appendChild(ul)
+  ul.appendChild(li1)
+  ul.appendChild(li2)
+  article.appendChild(table)
+  table.appendChild(tr)
+  tr.appendChild(td1)
+  tr.appendChild(td2)
+  tr.appendChild(td3)
+  article.appendChild(formEl)
+  formEl.appendChild(input)
+  formEl.appendChild(input2)
+
+  return {
+    section,
+    h1,
+    h2,
+    h3,
+    nav,
+    article,
+    ul,
+    li1,
+    li2,
+    table,
+    tr,
+    td1,
+    td2,
+    td3,
+    formEl,
+    input,
+    input2,
+  }
+}
+test('getRoles is a function', () => {
```

Comment:
I don't think we need this test.

Reply:
I don't disagree, but just for discussion the reason I include a test like that is that it distinguishes between problems of implementation and problems with configuration/plumbing.  If that test passes and the others fail then you know it's a problem with the implementation.  If that one fails along with the others then it lets you know at a glance there's some other problem (it's not being exported or something).  

But in this, and many cases it doesn't matter, and is mostly a way to get past the 'blank page' syndrome, a test to get started, haha.  

I'll pull them out.